modules/i3s/src/lib/parsers/parse-slpk/slpk-archieve.ts

```diff
@@ -1,6 +1,6 @@
 import {MD5Hash} from '@loaders.gl/crypto';
 import {FileProvider} from '@loaders.gl/loader-utils';
-import {parseZipLocalFileHeader} from '@loaders.gl/zip';
+import {IndexedArchive, parseZipLocalFileHeader} from '@loaders.gl/zip';
```

Comment:
File name typo `modules/i3s/src/lib/parsers/parse-slpk/slpk-archieve.ts` => `modules/i3s/src/lib/parsers/parse-slpk/slpk-archive.ts`

Reply:
Yeah. I didn't want to mix it up in the reivew. I want to change it in a separate PR